.pre-commit-config.yaml

```diff
@@ -60,3 +60,10 @@ repos:
       entry: from \.[\.\w]* import
       types: [python]
       language: pygrep
+    - id: no-internal-links
+      name: Check no internal links are in the docs
+      description: 'A quick check for the `docs.pymc.io` inside `docs/source/`'
+      files: ^docs/source/
+      entry: 'docs.pymc.io'
```

Comment:
is it possible to also check for aesara and arviz docs?

Reply:
Almost everything is possible 😉 . Should I include `aesara.readthedocs.io `and `arviz-devs.github.io`?

Reply:
Yes, that would be amazing, if it's not too difficult, can you add all websites in the intersphinx mapping?

https://github.com/pymc-devs/pymc/blob/main/docs/source/conf.py#L179

Reply:
Done ✔️